master/internal/rm/agentrm/scheduler_test.go

```diff
@@ -105,7 +105,9 @@ func newFakeAgentState(
 	if slotsUsed > 0 {
 		req := &sproto.AllocateRequest{
 			SlotsNeeded: slotsUsed,
-			Preemptible: true,
+			Preemption: sproto.PreemptionConfig{
+				Preemptible: true,
+			},
```

Comment:
`TimeoutDuration` will be 0. It looks like previously 1-hour was used as the default timeout? I also see that there's a check for `Preemption.TimeoutDuration != nil` ... I don't think that's possible. 

What is the intention use for a default / unset `TimeoutDuration`?

Reply:
this is just in the test, i can update it to use default timeout, but this test isn't impacted by timeout logic.

Reply:
I missed that - thanks for pointing it out!